advice_animal/runner.py

```diff
@@ -64,6 +65,14 @@ def run(
         if self.inplace:  # This is only applicable for `apply` command
             cur_cwd = os.getcwd()
             try:
+                os.chdir(repo)
+                try:
+                    run_cmd(["git", "diff", "-s", "--exit-code"])
+                except subprocess.CalledProcessError:
+                    raise ValueError(
+                        "Can't use inplace on a dirty checkout; commit first"
+                    ) from None
```

Comment:
What's the intent with `raise ValueError() from None`? 

Why suppress the context?

Reply:
It only suppresses the context down from here (subprocess internals), you still get to see the stack from you the caller to here.  I was originally thinking about having a `UserVisibleError` subclass that is caught and shown with no stack trace somewhere in the CLI, but will review them all in another PR.